### PR TITLE
Fix/fix typos login setup

### PR DESCRIPTION
**Pull request by shinji62**
Take a stab on the login vs log in and setup vs set up.

Only docs is modified

I used https://marketplace.visualstudio.com/items?itemName=valentjn.vscode-ltex to detect most of them.

The tools report a lot more issues but will take time to triage them

### CODE REVIEW

File: docs/content/preview/secure/enable-authentication/ysql.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ A YugabyteDB cluster with authentication enabled starts with the default admin u
 $ ./bin/ysqlsh -U yugabyte
 ```
 
-You are prompted to enter the password. Upon successful login to the YSQL shell, you should see the following output:
+You are prompted to enter the password. Upon successful log in to the YSQL shell, you should see the following output:
```
Comment (ddhodge) [inline docs/content/preview/secure/enable-authentication/ysql.md:98]:
```suggestion
You are prompted to enter the password. After logging in, you should see the following output:
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Enable YugabyteDB Anywhere authentication via LDAP
 headerTitle: Enable YugabyteDB Anywhere authentication via LDAP
-description: Use LDAP to enable login to YugabyteDB Anywhere.
+description: Use LDAP to enable log in to YugabyteDB Anywhere.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:4]:
```suggestion
description: Use LDAP for authentication in YugabyteDB Anywhere.
```

File: docs/content/stable/secure/enable-authentication/ysql.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ A YugabyteDB cluster with authentication enabled starts with the default admin u
 $ ./bin/ysqlsh -U yugabyte
 ```
 
-You are prompted to enter the password. Upon successful login to the YSQL shell, you should see the following output:
+You are prompted to enter the password. Upon successful log in to the YSQL shell, you should see the following output:
```
Comment (ddhodge) [inline docs/content/stable/secure/enable-authentication/ysql.md:98]:
```suggestion
You are prompted to enter the password. After logging in to the YSQL shell, you should see the following output:
```

File: docs/content/stable/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Enable YugabyteDB Anywhere authentication via LDAP
 headerTitle: Enable YugabyteDB Anywhere authentication via LDAP
-description: Use LDAP to enable login to YugabyteDB Anywhere.
+description: Use LDAP to enable log in to YugabyteDB Anywhere.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:4]:
```suggestion
description: Use LDAP for authentication in YugabyteDB Anywhere.
```

File: docs/content/v2.12/architecture/docdb-replication/async-replication.md [hunk 5]
```diff
@@ -58,7 +58,7 @@ This topology involves 1 source cluster sending data to many sink clusters. This
 This topology involves many source clusters sending data to one central sink cluster. This is currently not officially supported [#11535](https://github.com/yugabyte/yugabyte-db/issues/11535).
 
 ### More complex topologies
-Outside of our traditional 1:1 topology and the above 1:N and N:1, there are many other sensible configurations one might want to setup this replication feature with. However, none of these are currently officially supported. Some examples
+Outside of our traditional 1:1 topology and the above 1:N and N:1, there are many other sensible configurations one might want to set upthis replication feature with. However, none of these are currently officially supported. Some examples
```
Comment (ddhodge) [inline docs/content/v2.12/architecture/docdb-replication/async-replication.md:61]:
```suggestion
Outside of our traditional 1:1 topology and the above 1:N and N:1, there are many other sensible configurations one might want to use this replication feature with. However, none of these are currently officially supported. Some examples
```

File: docs/content/v2.12/secure/enable-authentication/ysql.md [hunk 6]
```diff
@@ -109,7 +109,7 @@ A YugabyteDB cluster with authentication enabled starts with the default admin u
 $ ./bin/ysqlsh -U yugabyte
 ```
 
-You will be prompted to enter the password. Upon successful login to the YSQL shell, you will see the following:
+You will be prompted to enter the password. Upon successful log in to the YSQL shell, you will see the following:
```
Comment (ddhodge) [inline docs/content/v2.12/secure/enable-authentication/ysql.md:112]:
```suggestion
You will be prompted to enter the password. After logging in to the YSQL shell, you will see the following:
```

File: docs/content/v2.14/architecture/docdb-replication/async-replication.md [hunk 7]
```diff
@@ -61,7 +61,7 @@ This topology involves many source clusters sending data to one central sink clu
 
 ### More complex topologies
 
-Outside of our traditional 1:1 topology and the above 1:N and N:1, there are many other sensible configurations one might want to setup this replication feature with. However, none of these are currently officially supported. Some examples:
+Outside of our traditional 1:1 topology and the above 1:N and N:1, there are many other sensible configurations one might want to set upthis replication feature with. However, none of these are currently officially supported. Some examples:
```
Comment (ddhodge) [inline docs/content/v2.14/architecture/docdb-replication/async-replication.md:64]:
```suggestion
Outside of our traditional 1:1 topology and the above 1:N and N:1, there are many other sensible configurations one might want to use this replication feature with. However, none of these are currently officially supported. Some examples:
```

File: docs/content/v2.14/secure/enable-authentication/ycql.md [hunk 8]
```diff
@@ -252,7 +252,7 @@ cassandra@ycqlsh> ALTER ROLE john WITH SUPERUSER = false;
 
 ### Enable and disable login privileges
 
-In the example above, you can verify that `john` is can login to the database by doing the following:
+In the example above, you can verify that `john` is can log in to the database by doing the following:
```
Comment (ddhodge) [inline docs/content/v2.14/secure/enable-authentication/ycql.md:255]:
```suggestion
In the example above, you can verify that `john` can log in to the database by doing the following:
```

File: docs/content/v2.16/secure/enable-authentication/ycql.md [hunk 9]
```diff
@@ -252,7 +252,7 @@ cassandra@ycqlsh> ALTER ROLE john WITH SUPERUSER = false;
 
 ### Enable and disable login privileges
 
-In the example above, you can verify that `john` is can login to the database by doing the following:
+In the example above, you can verify that `john` is can log in to the database by doing the following:
```
Comment (ddhodge) [inline docs/content/v2.16/secure/enable-authentication/ycql.md:255]:
```suggestion
In the example above, you can verify that `john` can log in to the database by doing the following:
```

**Review by ddhodge — APPROVED** [review]
Approve with minor changes

**Comment by bllewell** [comment]
This PR was merged even before I’d been alerted to the fact that I was an assigned reviewer. But that’s fine. Thanks for doing this work, Gwenn. Point taken that the tool you mentioned detects many other grammar issues that ideally should be fixed.

**Pull request merged**